azure-pipelines.yml

```diff
@@ -0,0 +1,85 @@
+# Python package
+# Create and test a Python package on multiple Python versions.
+# Add steps that analyze code, save the dist with the build record, publish to a PyPI-compatible index, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/python
+
+trigger:
+  tags:
+    include:
+      - '*'
+  branches:
+    include:
+      - master
+      - release/*
+      - refs/tags/*
+pr:
+  - master
+  - release/*
+
+jobs:
+  - job: pytest
+    # how long to run the job before automatically cancelling
+    timeoutInMinutes: 45
+    # how much time to give 'run always even if cancelled tasks' before stopping them
+    cancelTimeoutInMinutes: 2
+
+    pool: gridai-spot-pool
+
+    # ToDo: this need to have installed docker in the base image...
+    #container: "pytorchlightning/pytorch_lightning:base-cuda-py$[ variables['python.version'] ]-torch1.6"
+    container:
+      image: "pytorchlightning/pytorch_lightning:base-cuda-py3.8-torch1.6"
```

Comment:
Updated python version 3.7 to 3.8 just to match with the config in PL.
https://github.com/PyTorchLightning/pytorch-lightning/blob/db54b30776033d921fa4dc81c6fcdabc8fc29e48/.azure-pipelines/gpu-tests.yml#L39